lib/internal/errors.js

```diff
@@ -358,6 +358,9 @@ function message(key, args) {
     fmt = util.format;
     if (args === undefined || args.length === 0)
       return msg;
+    else if (require('internal/util/types').isUint8Array(args[0]))
```

Comment:
I am not sure if this is necessary, this would interfere with other formatters that might want their first arguments formatted differently. Also why are we coercing Uint8Arrays to empty strings? The message `Path  cannot end with / in operations invalid on directories` looks rather confusing, whereas `util.inspect` would print the Uint8Array properly if we just use `%s` in the formatter.

Reply:
I did this becuase you suggested it.
```js
if (isUint8Array(path) && path[path.length - 1] === 47) {
return handle(new errors.Error('ERR_INVALID_PATH_FORMAT', path));
```
> Note that this would be displayed as Path 102,105,108,101,58,47,47,47,116,109,112,47,116,101,115,116,47 ended with an invalid suffix character, we can either
1.Do not display the content in the path if it is a Uint8Array
>2. Decode it as UTF8 and display it
Personally I prefer 1 even though we currently assume the buffer is encoded in UTF8 in the C++ layer.

Reply:
@timotew Sorry, totally forgot about this, I think there is a third option: display the content as `util.inspect` displays it...still, come to think of it, `Path Uint8Array [1, 2, 3...] cannot end with / in operations invalid on directories` does look confusing as well. So the current message looks fine to me, but we should only change the formatter for this error instead of the message function.